app/konnect/dev-portal/audit-logging/webhook.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Set up an audit log webhook for Dev Portal
+content_type: how-to
+badge: enterprise
+---
+
+You can use the {{site.konnect_short_name}} UI or the Audit Logs API to configure webhooks for [audit logging](/konnect/dev-portal/audit-logging/). 
+
+{:.note}
+> **Note:** Currently, Dev Portal audit logs only support authorization logs, which are triggered when a user logs in to Dev Portal.
+
+{% include_cached /md/konnect/audit-logging/webhook-overview-prereq-siem-config.md %}
+
+
+## Create a webhook
+
+{% navtabs %}
+{% navtab Konnect UI %}
+1. From the navigation menu, open {% konnect_icon organizations %} **Organization**, then **Audit Logs Setup** and **Destinations**
+1. Fill in the fields in the **Destinations** tab. This allows you to set your audit log destination (the endpoint URL for your SIEM provider) and reuse it. 
+   * **Region endpoint**: The external endpoint that will receive audit log messages. 
+   * **Authorization Header**: The authorization type and credential to pass to your log collection endpoint. 
+    {{site.konnect_short_name}} will send this string in the `Authorization` header of requests to that endpoint.
+
+     For example, if you are setting up the webhook for Splunk, you could provide a Splunk access token: 
+     `"authorization":"Splunk example-token12234352535235"`.
+        
+    * **Log Format**: The output format of each log message. Can be CEF or JSON.
+    * **Skip SSL Verification**: Skip SSL verification of the host endpoint when delivering payloads.
+
+     {:.note}
+     > We strongly recommend not setting this to `true` as you are subject to man-in-the-middle and other attacks. This option should be considered only when using self-signed SSL certificates in a non-production environment.
+1. From the navigation menu, open {% konnect_icon Dev-portal %} **Settings**, then **Audit Logs Setup**.
+1. Fill in the fields in the **Setup** tab.
+    * **Audit log Destination**: select the destination that you want to use from the drop down list
+    * 
+1. Switch the toggle to `Enabled`, then save your webhook configuration.
+
+{% endnavtab %}
+{% navtab API %}
+
+Now that you have an external endpoint and authorization credentials, you can set up an audit log destination in {{site.konnect_short_name}}. The `/audit_log_destinations` endpoint allows you to set your audit log destination, which includes the endpoint URL and access key for your SIEM provider, and reuse it. 
+
+The {{site.konnect_short_name}} API uses [Personal Access Token (PAT)](/konnect/api/#authentication) authentication. You can obtain your PAT from the [personal access token page](https://cloud.konghq.com/global/account/tokens). The PAT must be passed in the `Authorization` header of all requests.
```

Comment:
@smritikjaggi This will be under the API tab

<img width="802" alt="Screenshot 2024-09-24 at 3 13 13 PM" src="https://github.com/user-attachments/assets/17b9161f-d704-41ef-9246-9bc066c218e8">